src/systemTest/java/randoop/main/RandoopSystemTest.java

```diff
@@ -89,83 +99,84 @@ public static void setupClass() {
-   *     are covered by all tests.  In this case, the test method will make a call like
-   *
-   *       generateAndTest(
-   *         testEnvironment,
-   *         options,
-   *         expectedRegressionTests,
-   *         expectedErrorTests);
-   *
-   *     where testEnvironment, packageName, regressionBasename, errorBasename, and options are all
-   *     defined in steps 1 and 2.
-   *     The expected-tests parameters are values of the ExpectedTests enumerated type.
-   *     Use the value SOME if there must be at least one test, NONE if there should be no tests,
-   *     and DONT_CARE if, well, it doesn't matter how many tests there are.
-   *     The generateAndTest() method handles the standard test behavior, checking the
-   *     standard assumptions about regression and error tests (given the quantifiers), and dumping
-   *     output when the results don't meet expectations.
-   *
-   *     By default, coverage is checked against all methods returned by Class.getDeclaredMethods()
-   *     for an input class. Some tests need to specifically exclude methods that Randoop should not
-   *     generate, or need to ignore methods.  These can be indicated by creating a
-   *     CoverageChecker object and adding these method names using either the exclude() or ignore()
-   *     methods, and then giving the CoverageChecker as the last argument to the alternate version
-   *     of generateAndTest(). When excluded methods are given, these methods may not be
-   *     covered, and, unless ignored, any method not excluded is expected to be covered.
-   */
+  * WRITING TEST METHODS:
+  *
+  * Methods with the @Test annotation will be run normally as JUnit tests.
+  * Each method should consist of one system test, and is responsible for setting up the
+  * directories for the test, setting the options for Randoop, running Randoop, compiling the
+  * generated tests, and then doing whatever checks are required for the test.  The steps each
+  * test should follow are:
+  *
+  * 1. Set up the test environment.
+  *
+  *    Each test method should create the working environment for running the test with a call like
+  *
+  *      SystemTestEnvironment testEnvironment = systemTestEnvironmentManager.createTestEnvironment(testName);
+  *
+  *    where testName is the name of your test (be sure that it doesn't conflict with the name
+  *    of any test already in this class).
+  *    The variable systemTestEnvironmentManager refers to the global environment for a run of the
+  *    system tests, and contains information about the classpath, and directories needed while
+  *    running all of the system tests.
+  *
+  * 2. Set the options for Randoop.
+  *
+  *    The method that executes Randoop takes the command-line arguments as a RandoopOptions object,
+  *    which can be constructed by the line
+  *      RandoopOptions options = RandoopOptions.createOptions(testEnvironment);
```

Comment:
I suggest that you create a single method that encapsulates both of these lines.  That will make clients a bit shorter, and more importantly it will make them less error-prone.